packages/dcos-integration-test/extra/util/python_test_server.py

```diff
@@ -286,7 +292,7 @@ def do_POST(self):  # noqa: ignore=N802
 # is to handle requests serially. If a connection is severed, this can result in the server hanging for the TCP
 # timeout before answering another requests...leading to failures.
 class ThreadingSimpleServer(ThreadingMixIn, HTTPServer):
-    pass
+    address_family = socket.AF_INET6
```

Comment:
@orsenthil @mellenburg Could you please review these changes as well? I've added ipv6 support to `python_test_server.py`